mount/mount_helper_unix.go

```diff
@@ -106,10 +110,18 @@ func ParseMountInfo(filename string) ([]MountInfo, error) {
 		if err != nil {
 			return nil, err
 		}
+		mm := strings.Split(fields[2], ":")
+		if len(mm) != 2 {
+			return nil, fmt.Errorf("parsing '%s' failed: unexpected minor:major pair %s", line, mm)
+		}
+		major, _ := strconv.Atoi(mm[0])
```

Comment:
Why not actually handle errors?

Reply:
Mirroring existing behavior that we may be depending on - https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/docker/docker/pkg/mount/mountinfo_linux.go#L52-L53

i haven't started at enough of these st_dev(s) to say if that's a good idea or a bad idea :)

Reply:
Also currently we don't throw an error if the string has bad contents ...